.env.example

```
@@ -230,6 +230,10 @@ SSO_GOOGLE_CLIENT_SECRET=
 # The domain that users must belong to in order to be able to log in.
 SSO_GOOGLE_HOSTED_DOMAIN=yourdomain.com
 
+# URLs for legal documents shown to new users during mobile Google SSO consent.
+LEGAL_TERMS_URL=
+LEGAL_PRIVACY_URL=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The LEGAL_PRIVACY_URL key should go before the LEGAL_TERMS_URL key

(UnorderedKey)